src/block/CodeBlock.ts

```diff
@@ -9,6 +9,7 @@ export interface CodeBlock {
   indent: number
   type: 'codeBlock'
   fileName: string
+  lang: string
```

Comment:
Language of CodeBlock is not simple.
For example, language of `code:cljc` is clojure, `code:processing` is java in Scrapbox.

This change only provides the file extension, and property name `lang` is not suitable.

Reply:
The behavior of `lang` is equal to `scrapbox.Page.lines`, so I thought it was natural to push a file extension into `lang`.
- For example:
- [![Image from Gyazo](https://i.gyazo.com/200993ad37521f63b7172396eb8b26f9.png)](https://gyazo.com/200993ad37521f63b7172396eb8b26f9)
But I also understand property name lang is not suitable. I'm going to close this PR.